### ISSUE TITLE
Setting frame width and frame heigh is only taking frame_width into account.

### ISSUE THREAD

**Issue by kolibril13** [ManimCommunity/manim#949]
When running this script:
```py
from manim import *
config["pixel_width"] = 100
config["pixel_height"] = 100
config["frame_width"] = 1
config["frame_height"] = 4
class Example(Scene):
    def construct(self):
        ar = Arrow(ORIGIN,2*UP)
        self.add(ar)
```
My expectation would be:  a 100x100 pixel image, where one can see the whole arrow (as it is in the field of view.)

Instead, I got this:

![Example](https://user-images.githubusercontent.com/44469195/105351674-25885300-5bed-11eb-8ad3-a7b85da60519.png)

A 100x100 pixel image, however not the whole arrow is visible.

This means, there is a problem with setting frame_height.

**Comment by huguesdevimeux** [comment]
Hmmm 
Isn't the arro too big to git in the frame?

**Comment by kolibril13** [comment]
Some more examples for this behavior:
https://flyingframes.readthedocs.io/en/latest/ch5.html#Pixel-Ratio-Unequal-to-16/9

**Comment by Timmmm** [comment]
Yeah I think I'm seeing the same problem. `frame_height` is just completely ignored.

Actually I investigated and that is the case. By default `frame_height` is ignored and the camera resizes maintaining aspect ratio and using `frame_width` to set the size.

See the code [here](https://github.com/ManimCommunity/manim/blob/94821c10d847b4b74982dd5ddd0646dd04d2121c/manim/camera/camera.py#L228).

It would be great if this were documented, particularly because all of the config examples use `frame_height` not `frame_width`!!

I also suggest the behaviour should change to this:

```
if frame_height is not None and frame_width is not None:
  resize(frame_width, frame_height)
elif frame_height is not None:
  resize(<preserve AR>, frame_height)
elif frame_width is not None:
  resize(frame_width, <preserve AR>)
else:
  default size
```
